tests/core/tests/test_admin_integration.py

```diff
@@ -647,6 +647,54 @@ def test_import_admin_uses_import_format_settings(self):
 
 
 class ExportAdminIntegrationTest(AdminTestMixin, TestCase):
+    def test_export_displays_resources_fields(self):
+        response = self.client.get("/admin/core/book/export/")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.context["fields_list"],
+            [
+                (
+                    "BookResource",
+                    [
+                        "id",
+                        "name",
+                        "author",
+                        "author_email",
+                        "imported",
+                        "published",
+                        "published_time",
+                        "price",
+                        "added",
+                        "categories",
+                    ],
+                ),
+                ("Export/Import only book names", ["id", "name"]),
+            ],
+        )
+
+        response = self.client.get("/admin/core/ebook/export/")
```

Comment:
I suggest we remove lines 675 to 695 - it seems like it tests the same thing as the previous block.

Reply:
ok